src/main/java/uc/doc/submitsolution/SubSDocDsRequestModel.java

```diff
@@ -2,24 +2,63 @@
 
 public class SubSDocDsRequestModel {
 
+    /**
+     * Name of the document
+     */
     private final String name;
 
+    /**
+     * The number of marks earned for this solution set
```

Comment:
Documenting instance variables for a class is not necessary, describing them in the constructor is sufficient I believe